packages/neos-ui-sagas/src/UI/Impersonate/index.js

```diff
@@ -4,7 +4,7 @@ import {actionTypes, actions} from '@neos-project/neos-ui-redux-store';
 import backend from '@neos-project/neos-ui-backend-connector';
 import {$get} from 'plow-js';
 
-export function * impersonateRestore({globalRegistry}) {
+export function * impersonateRestore({globalRegistry, routes}) {
```

Comment:
At this point, routes is undefined and therefore the whole UI is not working. For the impersonation Saga it works here we need another approach.

Reply:
```
bootstrap(globalRegistry, {store, frontendConfiguration, configuration, routes}))
```

That's the reason why it fails ;)

Reply:
oh sorry 😅